Mathlib/Topology/Algebra/Valued/ValuativeRel.lean

```diff
@@ -35,21 +35,54 @@ open ValuativeRel TopologicalSpace Filter Topology Set
 
 local notation "v" => valuation R
 
+lemma mem_nhds_iff (s : Set R) : s ∈ 𝓝 (0 : R) ↔
+    ∃ γ : (ValueGroupWithZero R)ˣ, { x | v x < γ } ⊆ s := by
+  convert IsValuativeTopology.mem_nhds (x := (0 : R))
+  rw [sub_zero]
+
+theorem hasBasis_nhds (x : R) :
+    (𝓝 (x : R)).HasBasis (fun _ => True)
+      fun γ : (ValueGroupWithZero R)ˣ => { z | v (z - x) < γ } := by
+  simp [Filter.hasBasis_iff, mem_nhds]
+
 variable (R) in
 theorem hasBasis_nhds_zero :
     (𝓝 (0 : R)).HasBasis (fun _ => True)
       fun γ : (ValueGroupWithZero R)ˣ => { x | v x < γ } := by
-  simp [Filter.hasBasis_iff, mem_nhds_iff]
+  convert hasBasis_nhds (0 : R); rw [sub_zero]
 
 @[deprecated (since := "2025-08-01")]
 alias _root_.ValuativeTopology.hasBasis_nhds_zero := hasBasis_nhds_zero
 
-variable [IsTopologicalAddGroup R]
+instance : ContinuousConstVAdd R R where
+  continuous_const_vadd x := continuous_iff_continuousAt.2 fun z ↦
+    ((hasBasis_nhds z).tendsto_iff (hasBasis_nhds (x + z))).2 fun γ _ ↦
+      ⟨γ, trivial, fun y hy ↦ by simpa using hy⟩
 
-theorem mem_nhds {s : Set R} {x : R} :
-    s ∈ 𝓝 x ↔ ∃ γ : (ValueGroupWithZero R)ˣ, { y | v (y - x) < γ } ⊆ s := by
-  simp only [← nhds_translation_add_neg x, ← sub_eq_add_neg, preimage_setOf_eq, true_and,
-    ((hasBasis_nhds_zero R).comap fun y => y - x).mem_iff]
+variable (R) in
+theorem tendsto_uncurry_add_nhds_zero :
```

Comment:
Similarly I don't think we want such a lemma. Spurious lemmas in the wrong generality like this could clog up the docs, mislead people In Loogle or `exact?` etc.